packages/i18n/locales/ko/translation.json

```diff
@@ -0,0 +1,441 @@
+    "imageURLTooltip": "이미지 링크를 삽입하시오. 예: https://moonphase.is/image.svg",
+    "initialSupply": "총 토큰 공급량",
+    "instantiatedAddress": "활성화된 주소",
+    "junoAddress": "Juno 주소",
+    "membersAddress": "회원의 주소",
+    "membersToAddOrUpdate": "추가/변경할 회원",
+    "membersToRemove": "제거할 회원",
+    "membershipBasedDescription": "회원이 남아 있을 가능성이 있는 작은 조직입니다. 기존 회원의 투표로 회원을 추가 및 제거할 수 있습니다.",
+    "membershipBasedTitle": "회원 전용",
+    "message": "메세지",
+    "migrateDescription": "이 기능은 선택하신 계약을 새로운 코드 ID로 <1>이동</1> 시킵니다.",
+    "migrateMessage": "메세지 이동",
+    "name": "이름",
+    "onlyMembersExecuteTitle": "회원에게 실행 권한 부여",
+    "onlyMembersExecuteTooltip": "본 기능이 활성화될 경우, 회원만이 통과된 제안을 실행할 수 있습니다",
+    "optional": "선택적",
+    "passingThresholdDescription": "정족수가 설정되어 있지 않은 경우, 이 비율은 제안이 통과되기 위해 필요한 ‘찬성’ 투표 비율입니다. 예를 들어, 통과 임계값을 50%로 설정하면 제안을 통과시키기 위해서 투표력의 절반이 제안에 찬성해야 합니다. 정족수가 설정되어 있는 경우, 통과 임계값은 유권자의 투표 결과로부터만 계산됩니다. 예를 들어 정족수 50% 및 통과 임계값이 50%인 경우, 전체 투표력의 25%가 '찬성'에 투표하였을 때 제안이 통과될 수 있습니다. 이 값의 설정을 권장합니다.",
+    "passingThresholdTitle": "통과 임계값",
+    "proposalDepositDescription": "제안을 생성하기 위해 예치해야하는 거버넌스 토큰 수 입니다. 이 설정을 통해 스팸성 제안을 방지할 수 있지만, 너무 높이 설정할 경우 제안 생성이 활발히 되지 않을 수 있습니다.",
+    "proposalDepositTitle": "제안에 필요한 보증금",
+    "proposalDescription": "제안에 대한 설명",
+    "proposalTitle": "제안의 제목",
+    "quorumDescription": "제안이 유효하기 위한 최소 투표력 비율입니다. 예를 들어 미국 하원의 경우 218명의 의원이 투표에 참석해야 합니다. 비활성 회원이 많은 DAO가 있는 경우 이 값을 너무 높게 설정하면 제안이 통과되기 어려울 수 있습니다.",
+    "quorumTitle": "투표 정족수",
+    "refundFailedProposalsDescription": "제안이 통과되지 못할 경우 제안자에게 보증금을 반환할 지에 대한 결정이 필요합니다. (통과된 제안은 항상 보증금을 돌려받습니다.) 특히 제안 보증금이 높은 경우, 이 기능을 활성화하면 회원들이 신중히 제안서를 작성하도록 유도할 수 있습니다.",
+    "refundFailedProposalsTitle": "통과되지 못한 제안에 대한 보증금 반환",
+    "refundFailedProposalsTooltip": "통과되지 못한 제안의 보증금을 반환하도록 하겠습니까?",
+    "requireProposalDepositTitle": "제안의 보증금 필요",
+    "requireProposalDepositTooltip": "이 기능이 활성화되면 회원은 제안을 생성하기 위해 보증금을 내야합니다.",
+    "selectAnImage": "이미지를 선택하십시오",
+    "smartContractAddress": "스마트 계약 주소",
+    "tierNameTitle": "티어 이름",
+    "tierNameTooltip": "회원의 \"소속\". 예시: \"코어 개발자\" or \"친구 또는 가족 \" 이 이름은 사용자의 편의성을 위해 라벨링용으로만 제공됩니다",
+    "tierVotingWeightPrompt_membershipBased": "이 티어의 회원에게는 {{weight}} 투표 가중치가 부여됩니다. 다른 투표 가중치를 가진 회원을 추가하시겠습니까? 새로운 티어를 추가합니다.",
+    "tierVotingWeightPrompt_tokenBased": "총 토큰 공급량 중, {{weight}}  만큼의 양이 해당 티어의 모든 회원에게 균등하게 분배됩니다. 다른 투표 가중치를 가진 회원을 추가하시겠습니까? 새로운 티어를 추가합니다.",
+    "tierVotingWeight_membershipBased": "회원별 투표 가중치",
+    "tierVotingWeight_tokenBased": "총 토큰 공급량 중 비율(%)",
+    "toValidator": "검증인에게",
+    "tokenAddress": "토큰 주소",
+    "tokenContractAddressTitle": "토큰 계약 주소",
+    "tokenDistributionTitle": "토큰 분배",
+    "tokenImage": "토큰 이미지",
+    "tokenInfo": "토큰 정보",
+    "unstakingPeriodDescription": "투표하기 위해, DAO의 회원들은 자신의 토큰을 DAO에 스테이킹해야 합니다. DAO를 탈퇴하거나 거버넌스 토큰으로 교환하려는 회원은 먼저 토큰을 언스테이킹해야 합니다. 이 설정은 DAO의 회원이 토큰을 언스테이킹한 후 해당 토큰을 활용하기 위해 대기해야 하는 시간을 설정합니다. 이 기간을 길게 설정할수록 토큰을 등록하는 사람들이 DAO 거버넌스에 더욱 참여하기를 원한다고 볼 수 있습니다.”,
```

Comment:
I think this needs to be replaced with a normal quote.

Reply:
I fixed !! Thank u :)